### PR TITLE
Added temporary :copy command

### PR DESCRIPTION
**Pull request by noughtmare**
This can be used while #843 is sorted out.

Pasting from clipboard can be done with `:set paste` and `<C-v>` (or `<C-V>` in the vty).

### CODE REVIEW

File: yi-keymap-vim/src/Yi/Keymap/Vim/Ex/Commands/Copy.hs [hunk 1]
```diff
@@ -0,0 +1,45 @@
...
+import           Yi.Buffer.Region                 (readRegionB)
+import           Control.Monad.Base               (liftBase)
+import           System.Hclip                     (setClipboard)
+import           Yi.Core                          (errorEditor)
+import           Yi.Editor                        (printMsg, withEditor)
+import           Data.Monoid                      ((<>))
+
+parse :: EventString -> Maybe ExCommand
+parse = Common.parse $ do
+    void (P.string "'<,'>copy")
```
Comment (jhance) [inline yi-keymap-vim/src/Yi/Keymap/Vim/Ex/Commands/Copy.hs]: In the `Commands.Common` there is a parser for regions like this! I think you should use that so we get a very general solution and even enable things like `:%copy`.

**Comment by jhance** [comment]
This looks pretty good to me. I'll play around with it tonight.

**Comment by noughtmare** [comment]
It doesn't work right with a block selection.
With the following selected:
```
testtest*t*est
test
testtest*t*est
```
(Asterisks represent the block selection)
It should copy:
```
tt
``` 
or 
```
t

t
```
But instead it copies:
```
test
test
testtest
```

**Comment by jhance** [comment]
MM yeah, I'm not sure how you would solve that issue.

**Comment by noughtmare** [comment]
This is also how it works in vim, so it might not be a bug.

**Comment by jhance** [comment]
Sounds like Vim also can't tell that its coming from visual mode.

**Pull request merged**